add-ons/che/che.addon.remove

```diff
@@ -1,6 +1,8 @@
 # Name: che
 # Description: Remove add on
 # Url: https://www.eclipse.org/che/docs/setup/openshift/index.html
+# Required-Vars: NAMESPACE
+# Var-Defaults: NAMESPACE=mini-che
 
 echo [CHE] Removing Image Stream
 oc delete -f streams/che-server-streams.json -n openshift
```

Comment:
@LalatenduMohanty  Required below changes at line 13-14.

```
echo [CHE] Removing #{NAMESPACE} project
oc delete project #{NAMESPACE}
```

Reply:
@budhrg It is already there.

Reply:
@budhrg Replaced teh hard coded value of `mini-che` by #{NAMESPACE}

Reply:
:+1: